mackup/utils.py

```diff
@@ -267,6 +267,23 @@ def get_copy_folder_location():
     return copy_home
 
 
+def get_icloud_folder_location():
+    """
+    Try to locate the iCloud Drive folder.
+
+    Returns:
+        (str) Full path to the iCloud Drive folder.
+    """
+    yosemite_icloud_path = '~/Library/Mobile Documents/com~apple~CloudDocs/'
```

Comment:
I don't trust python to be able to expand `~` in all envs, can you use the HOME env vars like in other get_*_location to be sure it works everywhere?